AccountService.js

```diff
@@ -90,25 +114,25 @@ export class AccountService {
   }
 
   /**
-   * @method setStatus
+   * Takes an id and a status string then changes an account's status.
+   * @memberOf AccountService
    * @param {Object} config
-   * @param {string} config.baseUrl
+   * @param {string} [config.baseUrl = /accounts]
    * @param {string} config.id - an account id
    * @param {string} config.status - a string that is either active or deleted
```

Comment:
I believe this is what we should have here:
```
@param {string(active|deleted)} config.status - The new account status.
```
Maybe need quotes around strings?  Hopefully this worksbased on this: https://stackoverflow.com/a/39259180

Reply:
good call man doing it now. btw thanks for looking up jsdoc so much

Reply:
I went ahead and made a TypeDef because we can re-use it in both bedrock-account and bedrock-account-http.
Maybe we should start a bedrock-jsdoc-types project and start creating type defs for common types?